doc/Reduction.md

```diff
@@ -0,0 +1,61 @@
+# Reduction
+
+The following is documentation for the Reduction kernel within `SYCL-BLAS`.
+
+## What is Reduction?
+
+`Reduction` is not a standard BLAS operation. It is provided as an extension in `SYCL-BLAS`. It refers to an operation where some or all elements of a matrix are reduced to a single scalar value via a binary operator. The supported operators are: 
```

Comment:
```suggestion
`Reduction` is not a standard BLAS operation.
It is provided as an extension in `SYCL-BLAS`.
It refers to an operation where some or all elements of a matrix are reduced to a single scalar value via a binary operator.
The supported operators are: 
```
Try to split lines in documentation, either by 80 columns, or by sentences (which is what I prefer for documentation). This makes the diffs smaller, making it easier to make changes in the future and also to make more targeted suggestions on PRs.

Reply:
I limited it to 80 columns since.